ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -371,12 +376,68 @@ def add_capacity(osd_size_capacity_requested):
     storageDeviceSets->count = (capacity reqested / osd capacity ) + existing count storageDeviceSets
 
     """
-
+    osd_size_existing = get_osd_size()
+    device_sets_required = int(osd_size_capacity_requested / osd_size_existing)
+    lvpresent = check_local_volume()
+    if lvpresent:
+        ocp_obj = OCP()
+        workers = get_typed_nodes(node_type='worker')
+        worker_names = [worker.name for worker in workers]
+        output = ocp_obj.exec_oc_cmd("get localvolume local-block -n local-storage -o yaml")
+        cur_device_list = output["spec"]["storageClassDevices"][0]['devicePaths']
+        path = os.path.join(constants.EXTERNAL_DIR, "device-by-id-ocp")
+        utils.clone_repo(constants.OCP_QE_DEVICEPATH_REPO, path)
+        os.chdir(path)
+        utils.run_cmd("ansible-playbook devices_by_id.yml")
+        f = open("local-storage-block.yaml", "r")
+        f2 = open("local-block.yaml", "w")
+        w1, w2, w3 = device_sets_required, device_sets_required, device_sets_required
```

Comment:
Please use more descriptive variable names. `f`, `f2`, `w1/2/3` don't tell anyone reading the code what they represent and it makes deciphering the code below much more difficult.

Reply:
Fixed it!